euth/follows/signals.py

```diff
@@ -0,0 +1,17 @@
+from django.db.models.signals import post_save
+
+from . import models
+
+
+def autofollow_hook(instance, **kwargs):
+    if hasattr(instance.project, 'id'):
+        models.Follow.objects.get_or_create(
+            project=instance.project,
+            creator=instance.creator,
+            defaults={
+                'enabled': True,
+            })
+
+
+post_save.connect(autofollow_hook, 'euth_comments.Comment')
+post_save.connect(autofollow_hook, 'euth_ideas.Idea')
```

Comment:
How is this going to work if follows is in core? Then the connectors also need to be added to the apps in the custom platforms, right?